article/sections/_7_latent_confounding.tex

```diff
@@ -0,0 +1,209 @@
+deconfounder algorithm \citep{wang_2019_blessings} in adjusting for unobserved 
+confounding. We use simulated mode choice data where travel distance 
+linearly confounds both travel time and travel cost. We then mask the travel 
+distance data and treat it as an unobserved variable. 
+
+
+We estimate three models:
+
+\begin{itemize}
+	\item Model 1: A multinomial logit with the correct original specification, 
+	except we ommit the travel distance variable in the specification without 
+	trying to adjust for it. This model represents the worst case scenario 
+	where a modeler ignores, or is unaware of, unobserved confounding.
+	\item Model 2: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance). In this method, we use all the variables in 
+	each mode's utility to recover that mode's confounder. This is in line 
+	with the approach taken in \citet{wang_2019_blessings}, where they use all the 
+	of observed variables in the factor model to recover a substitute 
+	confounder.
+	\item Model 3: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance), but this time, we only use travel time and 
+	cost in the factor model, instead of all the variables in the utility 
+	specification of each mode. By only using what we know are confounded 
+	variables to recover the substitute confounder, our goal is to analyze 
+	whether we can improve the accuracy of this approach by adopting a stronger 
+	prior on which variables are confounded. This can 
+	be in the form of building and testing candidate causal graphs that 
+	try to illustrate this confounding. 
+\end{itemize}
+
+
+We compare both the coefficient estimates on travel time and cost from 
+each of those three models to the true estimates used in the simulation. 
+We also compare the distribution of the recovered substitute confounder under each of 
+models 2 and 3 to the true confounder. The main findings of this exercise are the following:
+
+
+Using the true variables believed to be confounded (i.e. method 3 where only 
+travel time and cost are used to recover the confounder) leads to a better 
+recovery of the true confounder. Figure \ref{fig:qq-plot-method-2} and \ref{fig:qq-plot-method-3} show a QQ plot of the true 
+and recovered confounders under models 2 and 3 respectively. Looking at those 
+figures, we see that the distribution of the recovered substitute 
+confounder under method 3 is closer to that of the true confounder than method 
+2. This suggests that it may be better to run the deconfounder algorithm based 
+on a hypothesized causal graph, rather than just running it on all the 
+observed covariates. Please refer to Section \ref{sec:graph-construction} for how to build plausible causal graphs. 
+
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{qq-plot-method-2.png}
+   \caption{QQ plot of true confounder against recovered confounder using all covariates}
```

Comment:
For these plots we should show a line representing "ideal" or what we hope to recover. Should just be the straight 45-degree line I believe!

Reply:
That's too much spoon-feeding for me. Feel free to add it if you feel strongly about it, but we're just plotting true vs. estimated values. Anyone with some science background should be able to figure that out.

Reply:
Spoon-feeding done in #183.